CHANGELOG.md

```diff
@@ -158,15 +158,15 @@ The vpp-agent is now using custom VPP branch [stable-1801-contiv](https://github
     IP address set.
     - added support for virtio-based TAPv2 interfaces.
     - interface status is no longer stored in the ETCD by default and it can be turned on using appropriate
-    setting in defaultplugins.conf. See  [readme](plugins/vpp/README.md) for more details.  
+    setting in vpp.conf. See  [readme](plugins/vpp/README.md) for more details.  
```

Comment:
Not sure if we should be changing changelog's history. On one side, I stays same in the commit history anyway and on the other the actual date this particular change is relevant to was when it was still named defaultplugins. What do you think?

Reply:
I agree changelog history should not be edited in general. But we should be relevant, I mean no dead links, obsolete package names/files which does not exist anymore, etc.. 
Or, we can keep old package names in the changelog updating just links but then we need to mention the renaming as well, so everybody will find out why the package is not in the project anymore.

Reply:
Maybe in the parts where it isn't links use this format: